app/models/appeal_submission_upload.rb

```diff
@@ -4,4 +4,8 @@ class AppealSubmissionUpload < ApplicationRecord
   validates :decision_review_evidence_attachment_guid, :appeal_submission_id, presence: true
 
   belongs_to :appeal_submission
+
+  def form_attachment
+    @form_attachment ||= DecisionReviewEvidenceAttachment.find_by(guid: decision_review_evidence_attachment_guid)
```

Comment:
I feel like this can/should be able to be done in a more Rails-ey active record way... 

Like a 
```ruby
has_one :decision_review_evidence, :class_name => "DecisionReviewEvidenceAttachment", :foreign_key => "guid"
```

Or something along those lines rather than like this

Reply:
Totally. Thanks!